TESTING.md

```diff
@@ -0,0 +1,212 @@
+# Testing Guide for Latitude.sh Go Agent
+
+## Prerequisites
+
+1. **Server Requirements**:
+   - Ubuntu/Debian server with UFW firewall
+   - Root access or sudo privileges
+   - Go 1.22+ installed (for building from source)
+   - Active Latitude.sh account with firewall configuration
+
+2. **Required Environment Variables**:
+   ```bash
+   export LATITUDESH_BEARER="your_bearer_token_here"
```

Comment:
One other thing I just remembered: in our Terraform Provider, we use an environment variable named `LATITUDESH_AUTH_TOKEN`.

Should we keep a consistent naming pattern for all environment variables across the tools?